@ecomplus/storefront-template/template/js/netlify-cms/base-config/collections/layout/header.js

```diff
@@ -20,6 +20,16 @@ export default ({ baseDir, state }) => ({
           name: 'link',
           required: false,
           widget: 'string'
+        },
+        {
+          label: 'Cor de fundo da tarja',
+          name: 'bg_color_tarja',
```

Comment:
```suggestion
          name: 'background',
```

Ah tá que eu vou deixar você misturar português e inglês em nome de propriedade :grin: 
E já está dentro do objeto `marketing_stripe`, não precisa se referir a ele novamente no nome da propriedade, pode ser só `background`, no fim das contas ficará `marketing_stripe.background`, já é descritivo o suficiente...

Reply:
É fácil, até citei o código lá pros lojistas, mas o wesley é chato e como vi que era simples

Reply:
É simples, mas também é simples fazer o mesmo pro cabeçalho, e pro rodapé, e pra qualquer componente hahaha
Entendeu? E cada lojista vai achar que precisa de um, aí o ideal é que editem no CSS mesmo...
Ainda não acho interessante, mas se você quiser fixar os pontos aqui eu dou uma olhada no tópico e se pá damos merge.